protocol-units/bridge/config/src/lib.rs

```diff
@@ -17,6 +17,10 @@ pub struct Config {
 	/// Optional testing config
 	#[serde(default)]
 	pub testing: common::testing::TestingConfig,
+
+	/// Database configuration for the bridge indexer
+	#[serde(default)]
```

Comment:
I would call this `indexer_database` and the related module the same. This is just because there are other forms of persistence we've discussed already. But, otherwise, this is good.

Reply:
@l-monninger , I've committed the requested changes, let me know if that works